src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -20,11 +21,18 @@ public Board(){
     }
 
     private void initBoard(){
+    	playedMoves = 0;
+
         for (int i = 0; i < 3; i++)
             for(int j = 0; j < 3; j++)
                 board[i][j] = Player.NONE;
 
     }
+    
+    public void resetGame() {
+    	initBoard();
+    	winner = null;
```

Comment:
If you are using this function to initialize a new game, you should also initialize the currentPlayer variable. Right now when the user decides to play a new game, the current player is the one that played last in the previous game.
[This is a bug, please fix it]
```suggestion
    	winner = null;
    	currentPlayer = Player.X;
```

Reply:
This is addressed by commit: 9cada06701d6f9fcdbb1d934b562e3225483c716